README.md

```diff
@@ -17,4 +17,6 @@ Licensing
 
 Generally everything is LICENSE'D under the Apache 2 license by Abram Hindle.
 
-All text is licensed under the CC-BY-SA 4.0 http://creativecommons.org/licenses/by-sa/4.0/deed.en_US
\ No newline at end of file
+All text is licensed under the CC-BY-SA 4.0 http://creativecommons.org/licenses/by-sa/4.0/deed.en_US
+
+Added text.
```

Comment:
What's this?

Reply:
Was just testing whether new comment activities on git are being captured properly lol.